settings.coffee

```diff
@@ -312,6 +312,11 @@ if process.env["SHARELATEX_PASSWORD_VALIDATION_PATTERN"] or process.env["SHARELA
 # ShareLaTeX Server Pro
 #######################
 
+# Enables template thumbnail generation via Filestore.
```

Comment:
IIRC It's also used for the preview of PDFs in the file tree of a project.

-> which grants every user wild access on the conversion feature/vulnerability.

Reply:
Good catch @das7pad, I think we're going to change this so it's set by a new environment-variable, disabled by default. If admins are using CE/SP in a private setting they can turn it on explicitely.

Reply:
There is an environment variable for this already in place. See https://github.com/overleaf/filestore/commit/a441f46ed7fb2b958a6c642ddf64b53a38a0abf2